Interni/NormeProgetto/components/norme-c2.tex

```diff
@@ -0,0 +1,205 @@
+\section{Processi primari}
+\subsection{Fornitura}
+\subsubsection{Scopo}
+La fornitura rappresenta ciò che il fornitore svolge al fine di capire e soddisfare appieno quello che viene richiesto dal proponente.  In particolare,  il fornitore svolge un'analisi approfondita,  alla quale consegue la stesura dello \SdF{}, in modo da determinare criticità e rischi legati alla richiesta d'appalto. Inoltre, è necessario che venga definito un accordo sotto forma di contratto, così da regolarizzare la corrispondenza col proponente e tutto ciò che è legato col prodotto finale, tra cui consegna e manutenzione. A questo punto, si passa al determinare le procedure e le risorse necessarie,  sviluppandole nel \PdP{}. Quest'ultimo pone le basi per la realizzazione del prodotto, fino alla consegna finale.\newline 
```

Comment:
Cambiare
"La fornitura rappresenta ciò che il fornitore svolge ..."
in
"La fornitura è il processo che il fornitore utilizza ..." 
=======
"... alla quale consegue la stesura dello ..."
in
"... alla quale segue la stesura dello ..."
=======
"... regolarizzare la corrispondenza col proponente ..."
in
"... formalizzare la corrispondenza con il proponente ..."
=======
"... A questo punto, si passa al determinare le procedure e le risorse necessarie,  sviluppandole nel \PdP{} ..."
in
"... Vengono poi determinate e formalizzate nel \PdP{} le risorse e le procedure necessarie ..."

Reply:
Fixed